src/main/java/frc/robot/RobotContainer.java

```diff
@@ -1,346 +1,96 @@
+            drivetrain.setDefaultCommand(
+                    drivetrain.applyRequest(() -> drive.withVelocityX(-leftJoystick.getY() * MaxSpeedMetPerSec)
+                            .withVelocityY(-leftJoystick.getX() * MaxSpeedMetPerSec)
+                            .withRotationalRate(-rightJoystick.getX() * MaxAngularRateRadiansPerSec)));
+        } else {
+            drivetrain.setDefaultCommand(
+                    drivetrain.applyRequest(() -> drive.withVelocityX(-keyboardJoystick.getY() * MaxSpeedMetPerSec)
+                            .withVelocityY(-keyboardJoystick.getX() * MaxSpeedMetPerSec)
+                            .withRotationalRate(-keyboardJoystick.getX() * MaxAngularRateRadiansPerSec)));
+        }
 
-        masher.rightStickRight()
-            .onTrue(new InstantCommand(telescope::jogSetpointForward, telescope)); // jog telescope up
-        masher.rightStickLeft()
-            .onTrue(new InstantCommand(telescope::jogSetpointBackward, telescope)); // jog telescope down
+        rightJoystick.trigger()
+                .whileTrue(
+                        drivetrain.applyRequest(() -> driveRobot.withVelocityX(-leftJoystick.getY() * MaxSpeedMetPerSec)
+                                .withVelocityY(-leftJoystick.getX() * MaxSpeedMetPerSec)
+                                .withRotationalRate(-rightJoystick.getX() * MaxAngularRateRadiansPerSec)));
 
-        masher.rightStickUp()
-            .onTrue(new InstantCommand(wrist::jogSetpointForward, wrist)); // jog wrist right
-        masher.rightStickDown()
-            .onTrue(new InstantCommand(wrist::jogSetpointBack, wrist)); // jog wrist left
+        leftJoystick.trigger()
+                .whileTrue(drivetrain
+                        .applyRequest(() -> drive.withVelocityX(-leftJoystick.getY() * MaxSpeedMetPerSec * 0.5)
+                                .withVelocityY(-leftJoystick.getX() * MaxSpeedMetPerSec * 0.5)
+                                .withRotationalRate(-rightJoystick.getX() * MaxAngularRateRadiansPerSec * 0.5)));
 
-        // Intake
-        masher.leftTrigger()
-            .onTrue(new InstantCommand(intake::toggleIntake)); // toggle intake
+        rightJoystick.button(2)
+                .whileTrue(new InstantCommand(() -> drivetrain.seedFieldRelative()));
 
-        masher.rightTrigger()
-            .onTrue(new InstantCommand(intake::place)) // start place
-            .onFalse(new InstantCommand(intake::stop)); // stop place
+        controller.a().whileTrue(drivetrain.applyRequest(() -> brake));
+        controller.b().whileTrue(drivetrain
+                .applyRequest(
+                        () -> point
+                                .withModuleDirection(new Rotation2d(-controller.getLeftY(), -controller.getLeftX()))));
 
-        masher.back()
-            .onTrue(new HomeWrist(wrist));
+        controller.x().whileTrue(drivetrain.applyRequest(() -> drive.withIsOpenLoop(false)));
```

Comment:
My preference would be to put this into a new drive command (or multiple commands if needed). Also we can make our own deadband utility function and put it in a separate class so that can be used by the command

Reply:
Could this be a separate PR?